skyportal/models/tag.py

```diff
@@ -0,0 +1,33 @@
+import sqlalchemy as sa
+from sqlalchemy.orm import relationship
+
+from baselayer.app.models import AccessibleIfUserMatches, Base, join_model
+from skyportal.models import User
+from skyportal.models.obj import Obj
+
+
+class ObjTagOption(Base):
+    """A record of a list of tags that are available."""
+
+    name = sa.Column(sa.String, nullable=False, doc="Tags list.", unique=True)
+
+
+ObjTag = join_model("obj_tag", ObjTagOption, Obj)
+
+read = update = delete = AccessibleIfUserMatches("user")
+
+ObjTag.author_id = sa.Column(
+    "author_id",
+    sa.ForeignKey("users.id", ondelete="SET NULL"),
+    nullable=True,
+    index=True,
+    doc="ID of the user who created the tag association",
+)
+
+ObjTag.author = relationship(
```

Comment:
This syntax looks a bit different from other tables. We usually don't need to specify any foreign key as they are inferred from the author_id field you created above. Is that some exception due to the fact that we are adding these *after* creating the class? Let's double check, if this different syntax isn't required let's keep it consistent with the rest, such as: https://github.com/skyportal/skyportal/blob/main/skyportal/models/classification.py#L57-L63